src/components/tables/DataTable.vue

```diff
@@ -139,6 +139,7 @@ export interface Props {
    * When true, changing the items per page setting in one table will affect other tables.
    */
   globalItemsPerPage?: boolean;
+  singlePageSelect?: boolean;
```

Comment:
wouldn't applying this basically change the default behavior on all of our tables?

Wouldn't be better to change it to multi-page select and keep the single page select as the default behavior?

Reply:
okay, will update...

I once was on a page where the table selection span multiple pages

Reply:
Yes that was an exception though, I think most of the pages we have now are single by default (at least the ones with external pagination)